.repo/sample-list-generator/src/api-detector.ts

```diff
@@ -0,0 +1,144 @@
+import { IApiItem, TApiTypeResult, TExtensionApiMap } from './types';
```

Comment:
Question: do we need the type prefixes? I prefer to omit these to improve readability (and these are usually provided by the IDE automatically anyway).

Reply:
Agreed on removing these, it's not a pattern I've found particularly useful in the past :)